nfd-daemonset-combined.yaml.template

```
@@ -62,31 +62,27 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
-          image: k8s.gcr.io/nfd/node-feature-discovery:v0.6.0
+          image: k8s.gcr.io/nfd/node-feature-discovery-master:v0.6.0
```

```suggestion
          image: k8s.gcr.io/nfd/node-feature-discovery-master:v0.7.0
```
or v0.8.0 for the future case

---

Synced with latest master, using staging:master image